app/experimenter/docs/adrs/0003-analysis-visualization-architecture-approach.md

```diff
@@ -0,0 +1,49 @@
+# Analysis Visualization: Architecture Approach
+
+* Status: accepted
+* Author: Marina Samuel
+* Deciders: Nimbus Team
```

Comment:
To match the other ADRs you could combine author and deciders into `Deciders: Marina Samuel, Jared Lockhart, [whomever else is involved specifically, including folks that affect any ADR updates through discussions here]` but that's more of a suggestion than a must 🙂